scripts/qa_crowbarsetup.sh

```diff
@@ -3640,6 +3640,11 @@ function get_horizon
                         ['elements']['$horizon-server']"`
     horizonserver=`resolve_element_to_hostname "$element"`
     horizonservice=`resolve_element_to_hostname "$element" service`
+    horizonprotocol=http
+    `crowbar $horizon proposal show default | \
+        rubyjsonparse "
+                    puts j['attributes']['$horizon']['apache']['ssl']"` && \
```

Comment:
Thats hacky!
What if the value is not there and the `puts` fails (raises).
Rather make it explicit what you expect as output. This way the matching of the correct value is very hidden.